website/client/components/guilds/index.vue

```diff
@@ -0,0 +1,32 @@
+<template lang="pug">
+.row
+  secondary-menu.col-12
+    router-link.nav-link(:to="{name: 'tavern'}", exact, :class="{'active': $route.name === 'tavern'}") {{ $t('tavern') }}
+    router-link.nav-link(:to="{name: 'myGuilds'}", :class="{'active': $route.name === 'myGuilds'}") {{ $t('myGuilds') }}
+    router-link.nav-link(:to="{name: 'guildsDiscovery'}", :class="{'active': $route.name === 'guildsDiscovery'}") {{ $t('guildsDiscovery') }}
+
+  .col-12
+    router-view
+
+  group-form-modal
+</template>
+<!-- .col-md-2
+  button.btn.btn-primary(b-btn, @click="$root.$emit('show::modal','guild-form')") {{ $t('createGuild') }} -->
+
+<style>
+  body {
+    background-color: #f9f9f9;
```

Comment:
this background color for the body is covered in the global stylesheet https://github.com/HabitRPG/habitica/blob/develop/website/client/assets/scss/page.scss#L3